test/chpldoc/compflags/folder/save-sphinx/save-sphinx.prediff

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+(cd my-sphinx && \
+    echo * source/modules/*) > $2 || \
+        echo "ERROR cat-ing my-sphinx/* files" >> $2
+
+# Delete the dirs, so start_test does not descend into them...
+rm -rf docs/ my-sphinx/
```

Comment:
If we delete them, and something went wrong, we'll have to re-run the test by hand to see what happened.  It's better to have a .cleanfiles so that we know the test run won't be affected by a previous one.  Adding a NOTEST to the sphinx directories in this prediff script might be enough to prevent descending into them, but I could be wrong.